templates/base.html

```diff
@@ -32,7 +32,15 @@
                     <ul class="nav navbar-nav">
                     </ul>
                     <ul class="nav navbar-nav navbar-right">
-                        <li><a href="{% url 'users:signup' %}">Sign up</a></li>
+						{% if user %}
+							<li><a href="{% url 'users:profile' pk=user.pk %}">@{{ user.username }}</a></li>
```

Comment:
`{% url 'users:profile' pk=user.pk %}` throws NoReverseMatch at /users/signup/

Has this been tested?